vita3k/mem/src/mem.cpp

```diff
@@ -43,7 +43,14 @@ static void delete_memory(uint8_t *memory) {
 }
 
 static void alloc_inner(MemState &state, Address address, size_t page_count, Allocated::iterator block, const char *name) {
-    uint8_t *const memory = &state.memory[address];
+    Address alignment;
+#ifdef WIN32
+    alignment = 4096;
```

Comment:
```cpp
    int get_host_page_size() {
#if PLATFORM(WIN32)
        SYSTEM_INFO system_info = {};
        GetSystemInfo(&system_info);

        return system_info.dwPageSize;
#else
        return sysconf(_SC_PAGESIZE);
#endif
    }
```

I should forbid something like this. Also, use 4096. Then round them up to whatever Windows/Linux page size want.
The reason I tell you to do this is because the total of page was ordered to alloc was Vita page, and Vita page size is 4096. So the size Vita wants to allocate is 4096 * page_count. Then round them up to whatever Windows/Linux want,

Reply:
You should!

Any reason for `sysconf` over `getpagesize`? Just curious.

Reply:
Pretty sure I and petmac copy code from stackoverflow.